savant/meta/bbox.py

```diff
@@ -111,3 +126,40 @@ def scale(self, scale_x: float, scale_y: float):
         self.width = scaled_bbox[2]
         self.height = scaled_bbox[3]
         self.angle = scaled_bbox[4]
+
+    def polygon(self) -> np.ndarray:
+        """representation of the bbox using 4 corner points.
+        :return: polygon np.array([[x1, y1], [x2, y2], [x3, y3], [x4, y4]])
+        """
+        c, s = math.cos(self.angle / 180 * np.pi), math.sin(self.angle / 180 * np.pi)
```

Comment:
```suggestion
        angle_rad = self.angle / 180 * np.pi
        c, s = math.cos(angle_rad ), math.sin(angle_rad )
```

Cache degrees -> radians conversion